docs/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: The Dafny Language and Software Verification System
```

Comment:
By "Software Verification System" I meant the tools used to check/compile etc. Dafny programs. The project is not just language design -- it is tool development for that language.

Reply:
Fair enough, but IMO the main `dafny.org` front page should be primarily focussed on Dafny users rather than Dafny tool developers. We can have subpages dedicated to developing Dafny itself.

We could have something like "The Dafny Verified Programming Language" if we wanted to emphasize what makes Dafny unique.

Reply:
I agree fully that dafny.org is for users (the dafny GitHub project and wiki are for developers). But the users are using the tools, not just learning the language.